source/draw/gpu/GpuCanvasYuv420Semiplanar.ooc

```diff
@@ -76,9 +76,10 @@ GpuCanvasYuv420Semiplanar: class extends GpuCanvas {
 		uvLines free()
 	}
 	drawPoints: override func (pointList: VectorList<FloatPoint2D>) { this _target y canvas drawPoints(pointList) }
-	fill: override func {
-		this _target y canvas fill()
-		this _target uv canvas fill()
+	fill: override func ~color (color: ColorRgba) {
+		yuv := color toYuv()
+		this _target y canvas fill(ColorRgba new(yuv y, 0, 0, 255))
+		this _target uv canvas fill(ColorRgba new(yuv u, yuv v, 0, 255))
```

Comment:
I'm no color expert, so please explain why you take an `rgba` argument, convert it to `yuv`, then fill `y` and `uv` channels with `rgba` colors using the `yuv` components in red and green channels.